### ISSUE TITLE
Search results table not properly responsive

### ISSUE THREAD

**Issue by richardofsussex** [FreeUKGen/FreeBMD2#593]
On a screen less than about 865px wide, the advertisement begins to lie on top of the table.

At around 600px wide the table is getting clipped widthways (it does not scroll: see REG for the container needed to make such tables scroll).

**Comment by KristinaGadzhieva** [comment]
Where are we in this story?

**Comment by Vino-S** [comment]
Hi @KristinaGadzhieva 
Below are the screenshot of my current implementation:
<img width="784" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOG9XQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8bfaaa26cf5ba6aacb8eb18a416a661300d1d12d/image.png" alt="image.png" />

**Comment by Vino-S** [comment]
<img width="784" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODhXQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ecfc5193f9f7ea079010488c3aace7f071bc8ff1/image.png" alt="image.png" />

**Comment by KristinaGadzhieva** [comment]
@Vino-S Agreed: to make all links displayed as 'links' and single underlined (+ in story #609)

**Comment by richardofsussex** [comment]
Agreed to switch back to no underlining for links but to invent icons for the three left-most links in the bar above the table to pick them out.

**Comment by KristinaGadzhieva** [comment]
This story can be closed. Icons will be implemented in #609
